packages/vector_graphics_compiler/lib/vector_graphics_compiler.dart

```diff
@@ -18,9 +18,9 @@ import 'src/svg/theme.dart';
 import 'src/vector_instructions.dart';
 
 export 'src/_initialize_path_ops_io.dart'
-    if (dart.library.html) 'src/_initialize_path_ops_web.dart';
+    if (dart.library.js_interop) 'src/_initialize_path_ops_web.dart';
```

Comment:
We should ideally be consistent about how we switch on web compilation. @kevmoo / @ditman is there a preference between this and `io` (as was done in https://github.com/flutter/packages/pull/7991)?